# Pose with covariance
# orientation.x = vel_x
# orientation.y = vel_y

geometry_msgs/Pose pose
float64 xCov
float64 yCov
float64 vxCov
float64 vyCov
uint32 id

